reactor/presence.sls

```diff
@@ -1,6 +1,8 @@
+# actions to run when a minion disappears or appears in the master
+
 # remove unused keys for minions that are not present anymore
-{% for minion_id in data['lost'] %}
-remove_unused_keys_for_{{ minion_id }}:
-   wheel.key.delete:
-     - match: {{ minion_id }}
-{% endfor %}
+# {% for minion_id in data['lost'] %}
```

Comment:
Yeah, but the thing is that a minion that disconnects from the Master for, lets say, 10 minutes, will not be able to join again, and that is not good either...